testing/echo/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "echo"
```

Comment:
This name is going to clash with existing crates. I think even though we are not going to publish this to crates.io, we should use crate names that are at least likely to be unique (e.g. oak_echo). Or do you think we should not care about this problem at all, and pretend none of the crates in this repo will ever be published? Or should we add a tag / comment on the ones that are internal only?
cc @conradgrobler

Reply:
Done